docs/test.rst

```diff
@@ -73,9 +73,9 @@ Run QA Container
 Now run the QA container and mount the QA model repositories into the
 container so the tests will be able to access them::
 
-  $ nvidia-docker run -it --rm -v/tmp:/data/inferenceserver tensorrtserver_qa
+  $ nvidia-docker run -it --rm -v/tmp:/data/inferenceserver/<TRTIS_CONTAINER_VERSION> tensorrtserver_qa
```

Comment:
I don't think this is quite correct. The gen scripts will leave files in, for example, /tmp/qa_model_repository. There is no version directory there.

Reply:
Yes, there is no version directory in /tmp but the tests after modification will expect the models to be in /data/inferenceserver/<version>/qa_*  path. Hence, mounting the qa_* directories to the /data/inferenceserver/<version>.